app/views/layouts/application.html.erb

```diff
@@ -1,5 +1,9 @@
 <head>
-	<title>ProductMS</title>
+  <% if current_user && current_user.admin? %>
```

Comment:
Do we need to add `current_user` check as we already authentication user in application controller level?